### DISCUSSION TITLE
I’m making the animated series “Temple Legends” in toonsquid

### DISCUSSION THREAD

**Discussion by TempleLionStudios** [keiwando/toonsquid#595] [Show and tell]
https://youtu.be/F3X4che78-Q?si=9NCf_dwzzsuHxAqu

TEMPLE LEGENDS

A story about chosen family, treasure hunting, ancient temples, lost technology and spirituality... oh and a lot of video games!

**Comment by budaloops** [comment]
Sweet! I’m picking up Adventure Time vibes

**Reply by TempleLionStudios** [reply]
> Sweet! I’m picking up Adventure Time vibes

That's a great honor to be compared to Adventure Time! I'm also a big fan, so there's plenty of inspiration there for sure! Im also inspired by Regular Show and Studio Ghibli movies 😊

**Comment by ZafuMusic** [comment]
As you know I love everything about Temple Legends! <3
